services/order.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        tickets_ = []
+        user = get_user_model().objects.get(username=username)
+        order = Order(user=user)
+        if date:
+            date_transform = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            order.created_at = date_transform
+        else:
+            order.created_at = datetime.now()

```

Comment:
The `created_at` field should not be manually set with `datetime.now()`. It should rely on `auto_now_add=True` to automatically set the timestamp during order creation.